src/templates/rust-all/Makefile.tftpl

```diff
@@ -4,8 +4,10 @@ DOCKER_NAME      := arrow-${name}
 IMAGE_NAME       := ${name}
 BUILD_IMAGE_NAME := ghcr.io/arrow-air/tools/arrow-rust
 BUILD_IMAGE_TAG  := latest
+%{ if type == "svc" }
 DOCKER_PORT      := 8080
 HOST_PORT        := ${port}
+%{ endif }
```

Comment:
We don't need to run libraries with a network port, so remove it entirely. This will fix the leading space error as well.